### ISSUE TITLE
Changelog generation should be able to generate last change untagged

### ISSUE THREAD

**Issue by syrm** [bvieira/sv4git#17]
Hello,

Sometimes you want to generate the whole changelog until the last commit even if the commit is untagged.
So you can take this changelog add it to your repo, commit it, and tag the commit with the proper tag.

Exemple `https://github.com/syrm/test`
Generate changelog produce right now : 
```
# Changelog

## v3.0.0 (2021-04-10)

### Bug Fixes

- add 7 (573711f)
- add 6 (9422615)

---

## v2.3.1 (2021-04-10)

### Features

- add 5 (a28a0d8)

### Bug Fixes

- add 4 (c4d6748)

---

## v2.3.0 (2021-04-10)

### Bug Fixes

- add 2 (21c0d55)

---

## v1.0.0 (2021-04-10)

---
```

I think it would be cool if you add an option to be able to generate the changelog of the last commit too and produce something like this : 

```
# Changelog

## vX.X.X (2021-04-10)

### Features

- add 9 (0e02b6f)
- add 8 (31cce6f)

## v3.0.0 (2021-04-10)

### Bug Fixes

- add 7 (573711f)
- add 6 (9422615)

---

## v2.3.1 (2021-04-10)

### Features

- add 5 (a28a0d8)

### Bug Fixes

- add 4 (c4d6748)

---

## v2.3.0 (2021-04-10)

### Bug Fixes

- add 2 (21c0d55)

---

## v1.0.0 (2021-04-10)

---

With a changelog like that, it will be easy to only change X.X.X to v3.1.0, add this changelog into repository, commit it, and tag it v3.1.0

What do think about this ?

**Comment by bvieira** [comment]
I could add the result of `git sv rn` as the first item of changelog if there was any version change, something like:

**last tag:** 3.3.0

**release notes**

```bash
# release notes
$ git sv rn
## v3.4.0 (2021-04-10)

### Features

- add feature (2339fa3)
```

**changelog**

```bash
$ git sv cgl
# Changelog

## v3.3.0 (2021-04-10)

### Features

- update something (c711a8d)

---
```

**changelog + next release**

```bash
$ git sv cgl --add-next-version      
# Changelog

## v3.4.0 (2021-04-10)

### Features

- add feature (2339fa3)

---

## v3.3.0 (2021-04-10)

### Features

- update something (c711a8d)

---

```

But if the branch has no commits since last tag or only commits that doesn't bump the version, nothing will be added.

Is this ok?

**Comment by bvieira** [comment]
what happened with your repository? there is only one commit

https://github.com/syrm/test/commits

but the tags other commits

**Comment by syrm** [comment]
```
But if the branch has no commits since last tag or only commits that doesn't bump the version, nothing will be added.

Is this ok?
```
Seems good yes :-) 

About my repository i think i forget to push, but now i pushed

**Comment by bvieira** [comment]
flag `add-next-version` for changelog is available in version 2.2.0
